packages/core/test/state.test.ts

```diff
@@ -846,6 +846,44 @@ describe('State', () => {
 
       expect(executed).toBeFalsy();
     });
+
+    it('should return true when non-first parallel region changes value', () => {
+      const machine = createMachine({
+        type: 'parallel',
+        states: {
+          a: {
+            initial: 'a1',
+            states: {
+              a1: {
+                id: 'foo',
+                on: {
+                  // first region doesn't change value here
+                  EVENT: ['#foo', '#bar']
+                }
+              }
+            }
+          },
+          b: {
+            initial: 'b1',
+            states: {
+              b1: {},
+              b2: {
+                id: 'bar'
+              }
+            }
+          }
+        }
+      });
+
+      const { initialState } = machine;
+
+      expect(machine.transition(undefined, { type: 'EVENT' }).value).toEqual({
+        a: 'a1',
+        b: 'b2'
+      });
```

Comment:
Weirdly... this passes for me locally but fails on CI. Gonna investigate it on Monday